avocado/utils/archive.py

```
@@ -362,6 +362,10 @@ def extract(self, path="."):
                 )
 
         # Handle regular archives (zip and tar)
+        if self.is_tar:
+            self._engine.extraction_filter = getattr(
+                tarfile, "fully_trusted_filter", (lambda member, path: member)
+            )
         self._engine.extractall(path)
```

Ruff (0.16.1)

[error] Uses of `tarfile.extractall()`

(S202)